ui/package.json

```
@@ -13,7 +13,7 @@
   },
   "main": "index.js",
   "scripts": {
-    "build": "ember build",
+    "build": "DEBUG=ember-cli:* ember build",
```

```suggestion
    "build": "ember build",
```
Can this be reverted?